### PR TITLE
Added type hints to additional tests

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: Tests/test_imagedraw.py [hunk 1]
```diff
@@ -74,7 +74,14 @@ def test_mode_mismatch() -> None:
 
 @pytest.mark.parametrize("bbox", BBOX)
 @pytest.mark.parametrize("start, end", ((0, 180), (0.5, 180.4)))
-def test_arc(bbox, start, end) -> None:
+def test_arc(
+    bbox: tuple[tuple[int, int], tuple[int, int]]
+    | list[tuple[int, int]]
+    | list[int]
+    | tuple[int, int, int, int],
```
Comment (hugovk) [inline Tests/test_imagedraw.py:81]: This monster would be a good one to define as a type alias in `_typing.py`.

Comment (Yay295) [reply Tests/test_imagedraw.py:81]: Would `tuple[list[int], list[int]]` also be a valid option?

Comment (hugovk) [reply Tests/test_imagedraw.py]: Would that also be useful as an alias, for clearer re-use?

File: src/PIL/_typing.py [hunk 2]
```diff
@@ -15,4 +16,7 @@ def __class_getitem__(cls, item: Any) -> type[bool]:
                 return bool
 
 
+CoordList = Union[Sequence[float], Sequence[Sequence[float]]]
```
Comment (hugovk) [inline src/PIL/_typing.py:19]: Maybe just `Coords`? It's shorter, and a sequence can be a not-a-list such as a tuple:

```suggestion
Coords = Union[Sequence[float], Sequence[Sequence[float]]]
```

Comment (radarhere) [reply src/PIL/_typing.py]: Sure, I've made this change.

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**